nix/modules/scripts.nix

```diff
@@ -64,6 +64,17 @@
 
         git commit -m "chore(flakes): update $VERSIONS_DIR"
       '';
+
+      scripts-cargo-update = pkgs.writeShellScriptBin "scripts-cargo-update" ''
```

Comment:
this script doesn't ensure it has a `cargo` to call so IIUC the update command will fail, unless the runner ships a rust toolchain.

i think a simple approach would be to follow what we're doing with a similar script that we include [inline with the coreDev shell](https://github.com/holochain/holochain/blob/e7ffac490f8cbe3a5fc2397d3e3049581a4f0acd/nix/modules/devShells.nix#L132). you cold move this script there and then call it with `nix develop .#coreDev --command cripts-cargo-update`

Reply:
Good spot, moved as recommended